qa/smoke-test-http/src/test/java/org/elasticsearch/http/snapshots/RestGetSnapshotsIT.java

```diff
@@ -90,40 +90,45 @@ public void testSortOrder() throws Exception {
     }
 
     private void doTestSortOrder(String repoName, Collection<String> allSnapshotNames, SortOrder order) throws IOException {
-        final List<SnapshotInfo> defaultSorting = clusterAdmin().prepareGetSnapshots(repoName).setOrder(order).get().getSnapshots();
+        final boolean includeIndices = randomBoolean();
+        final List<SnapshotInfo> defaultSorting = clusterAdmin().prepareGetSnapshots(repoName)
+            .setOrder(order)
+            .setIndices(includeIndices)
```

Comment:
In my opinion both `indices` and `includeIndices` sounds quiet generic and might be confusing.
Should they be called `includeIndexNames`?